main.go

```diff
@@ -29,7 +29,7 @@ func setupFlags(app *kingpin.Application) {
 	app.Flag("config", "Load JSON configuration from file.").Action(loadConfig).String()
 	app.Flag("disable", "Disable previously enabled linters.").PlaceHolder("LINTER").Short('D').Action(disableAction).Strings()
 	app.Flag("enable", "Enable previously disabled linters.").PlaceHolder("LINTER").Short('E').Action(enableAction).Strings()
-	app.Flag("linter", "Define a linter.").PlaceHolder("NAME:COMMAND:PATTERN").StringMapVar(&config.Linters)
+	app.Flag("linter", "Define a linter.").PlaceHolder("NAME:COMMAND:PATTERN").Action(cliLinterOverrides).StringMap()
```

Comment:
Haha, nor am I :) Added the `StringMap()` call following the pattern I saw elsewhere in the code, e.g. right above this line –
```
app.Flag("disable", "Disable previously enabled linters.").PlaceHolder("LINTER").Short('D').Action(disableAction).Strings()
app.Flag("enable", "Enable previously disabled linters.").PlaceHolder("LINTER").Short('E').Action(enableAction).Strings()
```

Here's my understanding - If you pass in an argument in to these type functions (`Strings()`, `StringMap()`, etc), `kingpin`will do the CLI parsing for you, and set the values in the variable you passed in. If you don't pass in a var, it'll do the cli parsing (erroring if it's invalid), and discard the output. We don't care about the output as we're capturing it using the `Action()` call. i.e. main purpose of the additional `StringMap()` call is to declare intent to maintainers of the code.